modules/generator/processor/spanmetrics/spanmetrics.go

```diff
@@ -128,6 +168,118 @@ func (p *Processor) aggregateMetricsForSpan(svcName string, rs *v1.Resource, spa
 	p.spanMetricsDurationSeconds.ObserveWithExemplar(registryLabelValues, latencySeconds, tempo_util.TraceIDToHexString(span.TraceId), spanMultiplier)
 }
 
+// applyFilterPolicy returns true if the span should be included in the metrics.
+func (p *Processor) applyFilterPolicy(rs *v1.Resource, span *v1_trace.Span) bool {
+	// With no filter policies specified, all spans are included.
+	if len(p.Cfg.FilterPolicies) == 0 {
+		return true
+	}
+
+	for _, policy := range p.filterPolicies {
+		if policy.Include != nil {
```

Comment:
I'm not sure what `policyMatch()` should do in the case of a nil policy.  I agree it would be a little cleaner, but include is the inverse of exclude here, so for example returning a `true` value from `policyMatch()` isn't as clean.  Perhaps I can make exclude and include behave the same here, and then make the suggested adjustment.